common-components/index.md

```diff
@@ -178,37 +178,15 @@ __Further guidance__
 
 <a name="eapim"></a>
 ### API Management
-Enterprise integration and API management services make it easier to share data quickly and securely – whether you’re an integration partner, developer, publisher or data consumer.
+API management might be something that API developers and product owners in some DfE services would want to delegate to a central DfE team. Other services might want to run and manage their API in the same way they do their main service. Both are valid and supported. For more information on building API's at DfE see the [Technical Guidance](https://technical-guidance.education.gov.uk/guides/api-guidance/#content).
 
 **Enterprise API Management (EAPIM) Platform**
 
-The EAPIM platform provides a central place to host and discover APIs. It is a secure and fully accredited service that makes it easy to both publish and consume APIs.
```

Comment:
Iain McIntosh ok with editing down to be less salesy, but also keen not to go too far - suggest a discussion when we're back from leave, to iron out